### PR TITLE
RHDEVDOCS-2882-2: Tracker for Bug 1942908 - Include prerequisites sect…

### PR DESCRIPTION
**Pull request by rolfedh**
…ion when using clusterlogforwarder for syslog

Purpose: Suggest using a syslog server to receive syslog outputs from the cluster log forwarder. I did this by adding the phrase, "such as a syslog server," to various syslog topics. Also fix some typos and minor style issues.
 
- Aligned team: Dev Tools
- For branches: 4.5+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-2882
- Direct link to doc preview: https://deploy-preview-34587--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external?utm_source=github&utm_campaign=bot_dp#cluster-logging-collector-log-forward-syslog_cluster-logging-external
- SME review: not needed, wording change.
- QE reviewed by @anpingli 
- Peer review by @Srivaralakshmi 
- All reviews complete. Please merge now.

### CODE REVIEW

**Comment by TBD** [comment]
@rolfedh please use @ mentions correctly. thanks.

**Comment by rolfedh** [comment]
> @rolfedh please use @ mentions correctly. thanks.
Sorry, @TBD. I'll update the template I use to avoid spamming you.

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 1]
```diff
@@ -128,8 +128,8 @@ spec:
 <6> Configuration for an input to filter application logs from the `my-namespace` project.
 <7> Configuration for a pipeline to send audit logs to the secure external Elasticsearch instance:
 ** Optional. A name to describe the pipeline.
-** The `inputRefs` is the log type, in this example `audit`.
-** The `outputRefs` is the name of the output to use, in this example `elasticsearch-secure` to forward to the secure Elasticsearch instance and `default` to forward to the internal Elasticsearch instance.
+** The `inputRefs` are the log types that are inputs for this pipeline, such as `audit` in this example.
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:131]: The `inputRefs` are --> The `inputRefs` parameters are

Suggest to always follow the guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references

Or, you can write like this: Log types that are inputs for this pipeline, such as `audit` in this example.

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 2]
```diff
@@ -128,8 +128,8 @@ spec:
...
+** The `outputRefs` are the names of the outputs to use. For example, `elasticsearch-secure` forwards the audit log data to a secure Elasticsearch instance and `default` forwards the data to the internal Elasticsearch instance.
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:132]: The `outputRefs` are --> The `outputRefs` parameters are

Or: Names of the outputs to use.

Comment (Srivaralakshmi) [reply modules/cluster-logging-collector-log-forwarding-about.adoc:132]: - `elasticsearch-secure` forwards --> the `elasticsearch-secure` output forwards
- `default` forwards --> the `default` output forwards

Comment (Srivaralakshmi) [reply modules/cluster-logging-collector-log-forwarding-about.adoc:132]: Line 137: `my-app-logs` is the specific input for this pipeline.

Remark: my version is, "`my-app-logs` is the specific input for this Pipeline".

Reason behind this observation and suggestion: In the Pipelines section of the document, all the instance of the the term "pipeline" is written as "Pipeline" specially in the call out content of the YAML: https://docs.openshift.com/container-platform/4.7/cicd/pipelines/understanding-openshift-pipelines.html. Please check and incorporate the same style when it comes to pipeline content. You may want to talk/discuss with Preeti and Souvik, if the need be, to get some clarity on this.

Comment (Srivaralakshmi) [reply modules/cluster-logging-collector-log-forwarding-about.adoc:132]: - Line 138: 'default` is the output for this pipeline.

Follow the suggestion given above for Line 141 and 142 as well.

**Review by Srivaralakshmi — COMMENTED** [review]
@rolfedh Good work! Left some comments and suggestions. PTAL. Thanks!

**Pull request merged**